circle.py

```diff
@@ -11,14 +11,14 @@
 
 # Import libraries and set random number seed to 0
 import random
+import math
 random.seed(0)
 
 
 
 '''
  @brief     Function that samples a random location within a 2D bounding box.
- @details   A point is randomly and uniformally sampled in the bounding box 
-            provided in the input.
+ @details   A point is randomly and uniformally sampled in the bounding box             provided in the input.
```

Comment:
Fix spacing. According to the PEP8 style guide, there is an 80 character limit per line.